pkg/operator/operator.go

```diff
@@ -82,29 +82,36 @@ func New(config operatorconfig.Config, dnsManager dns.Manager, kubeConfig *rest.
 		return nil, fmt.Errorf("failed to create operator manager: %v", err)
 	}
 
+	// Create additional controller event sources from informers in the managed
+	// namespace. Any new managed resources outside the operator's namespace
+	// should be added here.
+	mapper, err := apiutil.NewDiscoveryRESTMapper(kubeConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get API Group-Resources")
+	}
+	operandCache, err := cache.New(kubeConfig, cache.Options{Namespace: "openshift-ingress", Scheme: scheme, Mapper: mapper})
+	if err != nil {
+		return nil, fmt.Errorf("failed to create openshift-ingress cache: %v", err)
+	}
+	buildStatusCache := cache.MultiNamespacedCacheBuilder([]string{"openshift-ingress-operator", "openshift-ingress"})
```

Comment:
This is a new feature of controller-runtime which will let us work with indexable caches across namespaces.